apps/speed.c

```diff
@@ -1130,6 +1165,127 @@ static int SM2_verify_loop(void *args)
 }
 #endif                         /* OPENSSL_NO_SM2 */
 
+static long kems_c[MAX_KEM_NUM][3];
```

Comment:
nit: add a comment about the : ` [3]  /* keygen, encaps, decaps */ `

Reply:
added in https://github.com/openssl/openssl/pull/19968/commits/d564ef96bab3c26de9652f743a1444f8f3015540